src/components/App/App.test.jsx

```diff
@@ -5,7 +5,7 @@ import { render, screen } from "@testing-library/react";
 describe("App", () => {
   it("renders the CYF Hotel header", () => {
     render(<App />);
-    const h1 = screen.getByRole("heading", { level: 1 });
-    expect(h1).toBeInTheDocument();
+    const headerElement = screen.getByRole("heading", { name: "CYF Hotel" });
+    expect(headerElement).toBeInTheDocument();
   });
```

Comment:
I found this approach interesting. The only thought that crossed my mind is: If we decide to change the text in <h1>, we will have to change the test too.
So, in this situation, I do not know which is better: Being more general using <h1> or being more specific using the proper title.